views/ddox.inc.function.dt

```diff
@@ -31,7 +32,7 @@
 			br
 			- foreach (size_t i, p; proto.parameters)
 				- auto pattribs = highlightDCode(getAttributeString(p.attributes, AttributeStringKind.normal));
-				- auto suffix = i+1 < proto.parameters.length ? "<span class=\"pun\">,</span>" : "";
+				- auto suffix = i+1 < proto.parameters.length ? "<span class=\"pun\">,</span>" : variadic_suffix;
```

Comment:
I just attached it to the last parameter, even though `, …` could arguably be put after a `br`, but that makes the formatting a lot more complex.